exporter/prometheusremotewriteexporter/helper.go

```diff
@@ -102,6 +102,21 @@ func sanitize(s string) string {
 	return s
 }
 
+func sanitizeLabels(s string) string {
+	if len(s) == 0 {
+		return s
+	}
+
+	// Note: No length limit for label keys because Prometheus doesn't
+	// define a length limit, thus we should NOT be truncating label keys.
+	// labels that start with _ are not sanitized
+	s = strings.Map(sanitizeRune, s)
+	if unicode.IsDigit(rune(s[0])) {
```

Comment:
Is this a recommended sanitization for prometheus labels that start w/ a digit? If so, could you add a link to the recommendation here.

Reply:
https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels

specifies that labels must match the regex [a-zA-Z_:][a-zA-Z0-9_:]* and label names beginning with __ are reserved for internal use.